### ISSUE TITLE
front end work for responses to a mySurvey survey

### ISSUE THREAD

**Issue by kmbeaulieu** [kmweinrich/SpartanSurveys#10]
- [ ] Have a non-moving section at the top that contains the survey title and description and a horizontal ruler bar thing
- [ ] Have a seperator between each response in the list
- [x] Have padding on the L/R side so it isn't up against the phone edge
- [x] Adjust text size if needed.

**Comment by SheethalSM** [comment]
i dont think it is necesary to have separator between response list as it is, separate like buttons,, atleast on my emulator?
but i dont think actual phones be any different?

**Comment by kmbeaulieu** [comment]
this is an expanded thing, not just a list of buttons. So it will have alllll the responses to eeevery question. it goes:
q1
r1
q2
r2
etc.

so between each response we need some sort of space or divider so it is more like 
Krystle's Response
q1
r1
q2
r2
...

---

Sheethal's Response
q1
r1
q2
r2
...

---

**Comment by SheethalSM** [comment]
ok cool...got it..i thought in between each question...my bad :)
